### DISCUSSION TITLE
Is there an option to disable full file details when giving temporary link to file?

### DISCUSSION THREAD

**Discussion by kuzyavrikus** [filebrowser/filebrowser#1341] [Q&A] [answered]
Hello, guys!
Is there an option to disable full file details when giving temporary link to file?

Asking because there is no reason for me to give all people who have temporary link full path-filename and so on information. Can i just leave download button and icon?

Using latest release (there was no file info in 2.5 release, only icon and icon plus QR in 2.6).
Just upgraded and see :)

Thanks!

![Screenshot 2021-03-18 at 10 31 40](https://user-images.githubusercontent.com/80192924/111596646-0b768580-87d6-11eb-8f7d-ac101423b61f.jpg)

**Comment by ramiresviana** [comment, marked as answer]
The path information displayed on the share view is not a full path, but relative to the share root. You can share a single file, when the root of the share points to a file it will have no location information to show.
